tests/fixtures/editions.py

```diff
@@ -0,0 +1,15 @@
+import json
+import pathlib
+import pytest
+
+@pytest.fixture
+def edition_data():
+    file_path = pathlib.Path("src/store/metadata/stub/content/editions.json")
+    with open(file_path, 'r') as json_file:
+        return json.load(json_file)["items"][0]
+    
+@pytest.fixture
```

Comment:
this wont work as the pytest.fixture decorator is already on the thing that's calling this. You can remove this.